### ISSUE TITLE
re: Ability to type before the first tag

### ISSUE THREAD

**Issue by guiramadasa** [xoxco/jQuery-Tags-Input#271]
Hi,

I have used this cool plugin in one of my projects. In that it needs to treat first tag as special. By default first tag becomes the primary tag as per the requirements. That part is fine.

But in case of a need of changing primary tag, i.e first tag, user have to remove all tags first and add tags again.

Instead is it possible have cursor in front (not always ideally, default way is fine, unless user needs to get cursor at begining). If such is possible, then it needs to reflect the same tag order in actual input values.

As another alternative, it will be cool if there is a way to drag and drop ordering for tags.

I am aware this is not a bug. But found no other contact with developers created this and using.

Thank you.

**Comment by jcspader** [comment]
Maybe we can use Drag Drop the tags to organize the sequence.

**Comment by imaustink** [comment]
Yes please use drag and drop! Also it would be awesome if you could use arrow keys to move to tags and edit them.

If you're looking for assistance with this project I would love to help. I don't really want to rebuild this from scratch when this already has 90% of what I need. I would rather extend this one. Either way I will probably fork the project so let me know.
